Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,7 +92,30 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
+            if (array == null) 
+            { 
+                return array; 
+            }
+            if ((index >= array.Length)||(index<0)) 
+            { 
+                return array; 
+            }
+            if (array.Length != 0)
+            {
+                int[] shortenedArray = new int[array.Length - 1];
+                for (int i = 0; i < shortenedArray.Length; i++)
+                {
+                    if (i >= index)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887772)